larpandoracontent/LArHelpers/LArPfoHelper.cc

```diff
@@ -173,6 +173,38 @@ void LArPfoHelper::GetAllDownstreamPfos(const ParticleFlowObject *const pPfo, Pf
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+void LArPfoHelper::GetAllDownstreamPfos(
+    const pandora::ParticleFlowObject *const pPfo, pandora::PfoList &outputTrackPfoList, pandora::PfoList &outputLeadingShowerPfoList)
+{
+    if (LArPfoHelper::IsTrack(pPfo))
+    {
+        outputTrackPfoList.emplace_back(pPfo);
+        const PfoList &childPfoList(pPfo->GetDaughterPfoList());
```

Comment:
Do we need this definition as opposed to using `pPfo->GetDaughterPfoList()` in the for loop?